### PR TITLE
feat(ui): support disabledFeatures, add nicer loading

### PR DESCRIPTION
**Pull request by psychedelicious**
- `disabledParametersPanels` -> `disabledFeatures`
- handle disabling `faceRestore`, `upscaling`, `lightbox`, `modelManager` and OSS header links/buttons
- wait until models are loaded to hide loading screen
- also wait until schema is parsed if `nodes` is an enabled tab
- secret loading screen dismissal button @ top right corner

### CODE REVIEW

File: invokeai/frontend/web/src/features/gallery/components/CurrentImageButtons.tsx [hunk 1]
```diff
@@ -328,24 +333,19 @@ const CurrentImageButtons = (props: CurrentImageButtonsProps) => {
...
-          title: t('toast.upscalingFailed'),
-          status: 'error',
-          duration: 2500,
-          isClosable: true,
-        });
-      }
+      handleClickUpscale();
+    },
+    {
+      enabled: () =>
```
Comment (maryhipp) [inline invokeai/frontend/web/src/features/gallery/components/CurrentImageButtons.tsx:339]: I may be misreading, but should this be `disabled`? All of those things listed below sound to me like they should disable the feature unless they're misnomers

Comment (psychedelicious) [reply invokeai/frontend/web/src/features/gallery/components/CurrentImageButtons.tsx:339]: whoops, ya

**Comment by psychedelicious** [comment]
This all needs to be refactored to support other config things like generation limits (size, steps, iterations, etc.). Marked as draft for now while I work on it.

**Comment by psychedelicious** [comment]
actually nvm, wanna get this and the img2img merged before working on the config stuff

**Review by blessedcoolant — APPROVED** [review]

File: invokeai/frontend/web/src/common/components/Loading/Loading.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  return (
+    <Flex
+      position="relative"
+      width="100vw"
+      height="100vh"
+      alignItems="center"
+      justifyContent="center"
+      bg="#151519"
+    >
+      <Image src={InvokeAILogoImage} w="8rem" h="8rem" />
```
[the remark this answers is not in the document]
Comment (psychedelicious) [reply invokeai/frontend/web/src/common/components/Loading/Loading.tsx:16]: added a spinner

**Pull request merged**